source/lexbor/dom/interfaces/document.h

```diff
@@ -153,6 +153,12 @@ lxb_dom_document_destroy_text(lxb_dom_document_t *document, lxb_char_t *text)
     return lexbor_mraw_free(document->text, text);
 }
 
+lxb_inline lxb_dom_element_t *
+lxb_dom_document_element(lxb_dom_document_t *document)
```

Comment:
@kostya 

Also, for this you need a separate patch.
Adding new functions to a document and serialization are different patches.